exercises/two-bucket/two_bucket_test.rb

```diff
@@ -36,6 +36,22 @@ def test_bucket_one_size_7_bucket_two_size_11_goal_2_start_with_bucket_two
     assert_equal 7, two_bucket.other_bucket
   end
 
+  def test_bucket_one_size_1_bucket_two_size_3_goal_3_start_with_bucket_two
+    skip
+    two_bucket = TwoBucket.new(1, 3, 3, 'two')
+    assert_equal 1, two_bucket.moves
+    assert_equal 'two', two_bucket.goal_bucket
+    assert_equal 0, two_bucket.other_bucket
```

Comment:
Should this be `assert_equal 1, two_bucket.other_bucket`?

Reply:
Please request mentoring on the exercise and discuss with a mentor.  This is a merged patch, and has survived 5 years, so it should probably be discussed at that level, and then escalated to an issue here, rather than on this accepted patch.